CDS/scripts/js_utils/youtube/youtube-upload-lib.js

```diff
@@ -131,24 +132,42 @@ function addPosterImageIfExists(connection, videoId, youtubeClient, account) {
 
 function uploadToYoutube(connection) {
     return new Promise((resolve, reject) => {
+        console.log("DEBUG: getting auth client");
         youtubeAuth.getAuthClient(connection).then(authClient => {
+            console.log("DEBUG: got auth client.  Getting youtube client for version " + YOUTUBE_API_VERSION);
             const ytClient = googleapis.youtube({version: YOUTUBE_API_VERSION, auth: authClient});
 
             this.getYoutubeData(connection).then(ytData => {
+                console.log("DEBUG: got youtube data " + JSON.stringify(ytData));
+                console.log("Attempting to insert video...");
                 ytClient.videos.insert(ytData, (err, result) => {
                     if (err) {
+                        console.error("ERROR: unable to insert YT video: " + err);
                         reject(err);
+                    } else {
+                        console.log("Video added successfully.  Adding poster frame...");
+                        addPosterImageIfExists(connection, result.id, ytClient, ytData.onBehalfOfContentOwner)
+                            .then(() => {
+                                dataStore.set(connection, 'meta', 'youtube_id', result.id)
+                                    .then(() => {
+                                        resolve(result);
+                                        console.log("SUCCESS: added poster frame");
```

Comment:
could we move this message to the `addPosterImageIfExists` function as that promise resolved if there is no poster image set on the atom.